Formula/xcode-kotlin.rb

```diff
@@ -0,0 +1,56 @@
+class XcodeKotlin < Formula
+  desc "Kotlin Native Xcode Plugin"
+  homepage "https://github.com/touchlab/xcode-kotlin"
+  url "https://github.com/touchlab/xcode-kotlin.git", tag: "1.2.0", revision: "3af7e5d921c3756a3adc83ad504ac0db2a5643e8"
+  license "Apache-2.0"
+  head "https://github.com/touchlab/xcode-kotlin.git", branch: "main"
+  depends_on "openjdk" => :build
+  depends_on xcode: ["12.5", :build]
+  depends_on :macos
+
+  def install
+    ohai "Deciding which architecture to build"
+    arch = `uname -m`.strip
```

Comment:
Formula should not output anything.

Reply:
Do you mean I should remove all the `ohai`? Or was this only about the usage of the uname? Thanks!

Reply:
All `ohai`, `odie` etc